merkle/rfc6962/rfc6962.go

```diff
@@ -15,30 +15,38 @@
 // Package rfc6962 provides hashing functionality according to RFC6962.
 package rfc6962
 
-import "crypto"
+import (
+	"crypto"
+	_ "crypto/sha256" // SHA256 is the default algorithm.
+)
 
 // Domain separation prefixes
 const (
 	RFC6962LeafHashPrefix = 0
 	RFC6962NodeHashPrefix = 1
 )
 
-// Hasher is a SHA256 based TreeHasher.
-var Hasher = &TreeHasher{Hash: crypto.SHA256}
+// DefaultHasher is a SHA256 based LogHasher.
+var DefaultHasher = New(crypto.SHA256)
 
-// TreeHasher implements the RFC6962 tree hashing algorithm.
-type TreeHasher struct {
+// RFCHasher implements the RFC6962 tree hashing algorithm.
```

Comment:
This should probably be called RFC6962Hasher. Or it will be confusing when we have v2 of the RFC.